backend/.gitignore

```diff
@@ -2,4 +2,5 @@ venv
 db.sqlite3
 __pycache__
 .env
-mydatabase
\ No newline at end of file
+mydatabase
+myvenv
```

Comment:
might be better to just name local virtual environment to `venv` and database to `db.sqlite3`, so .gitignore content won't be changed for same thing.